k8s/spack/spackbotdev-spack-io/deployments.yaml

```diff
@@ -95,27 +101,35 @@ spec:
       serviceAccountName: spackbotdev-spack-io
       containers:
       - name: worker
-        image: "ghcr.io/spack/spackbot-workers:latest"
+        #image: "ghcr.io/spack/spackbot-workers:latest"
         # Can use some other image for testing, see README.md
-        # image: "ghcr.io/scottwittenburg/spackbot-workers:0.0.93"
+        image: "ghcr.io/kwryankrattiger/spackbot-workers:0.0.1"
         imagePullPolicy: Always
         resources:
           requests:
             cpu: 900m
             memory: 1G
         # Mount secrets to non-existing location
         volumeMounts:
-          - mountPath: "/git_rsa"
-            name: spack-bot-dev-idrsa
-            readOnly: true
+        - mountPath: "/git_rsa"
```

Comment:
I'm not sure whether indented or not indented is preferred here.  But somehow I managed to end up with the indentation not matching between the production and development versions.  Can you either add indents here, or remove them in the dev version?